src/main/kotlin/blackjack/domain/player/Player.kt

```diff
@@ -0,0 +1,30 @@
+package blackjack.domain.player
+
+import blackjack.domain.Score
+import blackjack.domain.card.CardDeck
+import blackjack.domain.player.vo.Name
+
+class Player private constructor(
+    val name: Name,
+    val cardsInHand: CardsInHand
+) {
+    var isStay: Boolean = false
+        get() = field || cardsInHand.calculateScore().isGreaterThan(Score.BLACKJACK)
```

Comment:
isStay는 외부에서도 set을 할 수 있습니다.
외부에서 변경을 막기 위해서는 private set을 붙여주시면 됩니다.
```suggestion
    var isStay: Boolean = false
        get() = field || cardsInHand.calculateScore().isGreaterThan(Score.BLACKJACK)
        private set
```

Reply:
반영해두겠습니다 😄

Reply:
추가해두겠습니다 ~ 😄